spdx-utils/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -321,5 +322,19 @@ class SpdxExpressionTest : WordSpec() {
                 spdxExpression.disjunctiveNormalForm() shouldBe dnf
             }
         }
+
+        "valid choices" should {
```

Comment:
Again, use the function name? Or "Valid choices" (capitalized)?

Reply:
Done.